packages/ui/src/components/data-display/chip/index.stories.tsx

```diff
@@ -1,21 +1,20 @@
 import React from "react";
-import { ComponentMeta, Story } from "@storybook/react";
+import type { Meta, StoryObj } from "@storybook/react";
```

Comment:
Is the `import type` needed here?

Reply:
Yeah. Actually, we should start using `import type` always when importing types. The actual import will be erased at runtime and only used at compile time, resulting in a potential reduction of bundle size.